eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -126,7 +126,7 @@ stages:
           jdkVersionOption: $(JavaTestVersion)
 
         PreRunSteps:
-        - template: ../steps/cosmos-emulator.yml
+        - template: ../../../common/pipelines/templates/steps/cosmos-emulator.yml
```

Comment:
For these imports we generally try to use absolute paths from the root such as `/eng/common/pipelines/templates/steps/cosmos-emulator.yml` so it is more clear where they are coming from and a little cleaner.